EU4toV2/Source/EU4World/Provinces/Provinces.h

```diff
@@ -1,36 +1,36 @@
 #ifndef PROVINCES_H
 #define PROVINCES_H
-
+#include "Configuration.h"
 #include "EU4Province.h"
 #include "Parser.h"
-#include "../../Configuration.h"
 #include <map>
 
-namespace mappers {
-	class SuperGroupMapper;
+namespace mappers
+{
+class SuperGroupMapper;
 }
 
 namespace EU4
 {
-	class Regions;
+class Regions;
+class Provinces: commonItems::parser
+{
+  public:
+	explicit Provinces(std::istream& theStream);
 
-	class Provinces: commonItems::parser
-	{
-	public:
-		explicit Provinces(std::istream& theStream);
-		
-		[[nodiscard]] auto geTotalProvinceWeights() const { return totalProvinceWeights; }
-		[[nodiscard]] const auto& getAllProvinces() const { return provinces; }
-		std::shared_ptr<Province> getProvince(int provinceNumber);
-		
-		void determineTotalProvinceWeights(const Configuration& configuration);
+	[[nodiscard]] auto geTotalProvinceWeights() const { return totalProvinceWeights; }
```

Comment:
geTotalProvinceWeights -> getTotalProvinceWeights

Reply:
Always wondered about that one.
https://github.com/ParadoxGameConverters/EU4toVic2/commit/a22ae045aba36a33266870549121a3fce6533f61
Figured you wanted it to stand out or something. Would be a shame to change it, now that it's become bedrock of converter history.

Reply:
Ha! Should have known it was my own fault. It usually is.